pyserini/pyclass.py

```diff
@@ -57,6 +57,8 @@
 JDocument = autoclass('org.apache.lucene.document.Document')
 JSearcher = autoclass('io.anserini.search.SimpleSearcher')
```

Comment:
Should we rename to `JSimpleSearcher` and `JSimpleNearestNeighborSearcher` just to make everything consistent?

Reply:
sure, good point, done.